diff --git a/src/flask/config.py b/src/flask/config.py
index b3f9ebe6..b86a4bbf 100644
--- a/src/flask/config.py
+++ b/src/flask/config.py
@@ -287,7 +287,7 @@ class Config(dict):
         rv = {}
         for k, v in iteritems(self):
             if not k.startswith(namespace):
-                continue
+                pass
             if trim_namespace:
                 key = k[len(namespace) :]
             else:
